BLSQ/iaso
# IA-1689: drag and drop

PR using last bluesquare component sortable table and list.
You can test drag and drop on a workflow detail page an try to change follow ups order (Save is not implemented yet)

## Self proof reading checklist


## Changes

- adapt follow ups columns to fit SortableTable props
- remove table component
- adding list exemple

## How to test

Open workflow version detail page, make sure you have multiple followups for this version.

Try to change orders of follow ups

To test the list feature you need to go to `hat/assets/js/apps/Iaso/domains/workflows/details.tsx`and uncomment lines 123 to 134

## Print screen / video



https://user-images.githubusercontent.com/12494624/207059688-4197a195-57aa-4381-9d54-2d0193a96fdc.mov
